src/screens/poa-form/index.js

```diff
@@ -267,8 +272,16 @@ class PoAForm extends React.Component {
       />
     )
 
+    // Hide the disclaimer if `acceptedModal` is true
+    const disclaimer = (
```

Comment:
Is it better to handle this by conditionally rendering the component or use CSS? Part of the answer might be taking into animation into consideration and if we want to do that with CSS or use the Grommet animation api.

Reply:
With react-modal already take care of the animation and style for the modal to behave as a modal.

Reply:
`Layer` is Grommet's built in modal component - I'm happy enough with it.

Grommet's [Animate utility](http://grommet.io/docs/animate) looks intuitive enough, I'll use that instead of conditionally rendering.